x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/predicate/operator/arithmetic/Neg.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.esql.expression.predicate.operator.arithmetic;
+
+import org.elasticsearch.compute.ann.Evaluator;
+
+public class Neg {
```

Comment:
Negate. We aim for clarity (also avoid a slang term).

Reply:
I prefer `Negate` as well! However, that would be inconsistent with the [corresponding expression](https://github.com/elastic/elasticsearch/blob/feature/esql/x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/arithmetic/Neg.java) from `ql`. The evaluators defined inside the [esql expression subpackage](https://github.com/elastic/elasticsearch/tree/feature/esql/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/) seem to follow the structure of the [ql expression subpackage](https://github.com/elastic/elasticsearch/tree/feature/esql/x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression) and this would break the pattern.

I could either
1. live with the inconsistency that the evaluator for `Neg` expressions is called `Negate`,
2. stay consistent, rename both to `Negate` (though that would create a bit of noise - but could be done in a follow-up PR), or
3. stick with `Neg`.

What do you think?

Reply:
go with 3.